### DISCUSSION TITLE
How to set filters?

### DISCUSSION THREAD

**Discussion by tonyrulez** [FreshRSS/FreshRSS#3621] [Q&A]
I would like to set filters to auto-star an article so when scrolling through the feed I can see immediately which feeds I'm interested in. For instance if article name contains XYT then mark as favourite. Can't find this feature but it must be somewhere :)

**Comment by aledeg** [comment]
It's not available yet. It's planned for the near future though

**Reply by tonyrulez** [reply]
That's appreciated. There should be a delete action too for the filters, I'm using RSS-Bridge which sometimes throws an error as an article.
